Corona-Warn-App/src/main/java/de/rki/coronawarnapp/appconfig/mapping/CovidCertificateConfigMapper.kt

```diff
@@ -51,7 +53,7 @@ class CovidCertificateConfigMapper @Inject constructor() : CovidCertificateConfi
 
     data class CovidCertificateConfigContainer(
         override val testCertificate: CovidCertificateConfig.TestCertificate = TestCertificateConfigContainer(),
-        override val expirationThresholdInDays: Int = 1
+        override val expirationThresholdInDays: Duration = Duration.standardDays(1),
```

Comment:
should not be `14` ? https://github.com/corona-warn-app/cwa-app-tech-spec/blob/78009f7e3aa6aef56892caedcfe478bfb9d4d691/docs/spec/handling-of-app-config.md#parameters-for-digital-green-certificate

Reply:
Done and added tests.